src/dir.targets

```diff
@@ -160,8 +147,35 @@
 
   <!-- On command line run unit tests on CoreCLR after the Test target -->
   <Target Name="RunTestsForProject"
-          AfterTargets="Test"
-          Condition="'$(RunTestsForProject)'=='true'">
+          AfterTargets="CheckTestCategories"
```

Comment:
Please add back AfterTargets="Test" and add DependsOnTargets="CheckTestCaegories"